spec/bundle/vscode_extension_installer_spec.rb

```diff
@@ -27,9 +27,9 @@
         allow(described_class).to receive(:installed_extensions).and_return(["foo"])
       end
 
-      it "skips" do
+      it "skips ignoring case" do
```

Comment:
Thanks for tweaking the test! In this case: I think it'd be worth adding a new test rather than adjusting this one so we make sure both paths work.

Reply:
done in latest commit